systemd/systemd.go

```diff
@@ -86,7 +86,9 @@ var JournalctlCmd = jctl
 type Systemd interface {
 	DaemonReload() error
 	Enable(service string) error
+	EnableNow(service string) error
```

Comment:
Good idea indeed. I'll merge just because we need this now, but we should revisit.